lib/ApiRequestor.php

```diff
@@ -54,17 +54,13 @@ private static function _encodeObjects($d)
      */
     public function request($method, $url, $params = null, $headers = null)
     {
-        if (!$params) {
-            $params = array();
-        }
-        if (!$headers) {
-            $headers = array();
-        }
+        $params = $params ?: [];
+        $headers = $headers ?: [];
```

Comment:
I opportunistically did a few of these "fixes" which is why the PR has a negative net line count.

Reply:
+1.